Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -22,8 +22,135 @@ private static void Test(string args, int expected)
 
         public static int FindDigit(string equation)
         {
-            // Add your code here.
-            throw new NotImplementedException();
+            // a * b = c
+            string a = "";
+            string b = "";
+            string c = "";
+            int n = equation.Length;
+            int i = 0;
+            int ques_mark = 0;
+            //Extrct 3 numbers
+            //first number
+            while (i < n)
+            {
+                if (equation[i] == '?') ques_mark = 1;
+                if (equation[i] == '*')
+                {
+                    i++;
+                    break;
+                }
+                else
+                {
+                    a += equation[i];
+                }
+                i++;
+            }
+            //second number
+            while (i < n)

```

Comment:
Using so many loops is never an optimal solution try to minimize the use of loops and find an optimal solution.
Hint: go through the core functions of string